app/src/main/java/com/codingblocks/cbonlineapp/mycourse/content/CourseContentFragment.kt

```diff
@@ -442,4 +443,14 @@ class CourseContentFragment : BaseCBFragment(), AnkoLogger, DownloadStarter {
             }
         }
     }
+
+    private fun isMyServiceRunning(serviceClass: Class<*>): Boolean {
```

Comment:
@championswimmer is this is a good way to handle this?

Reply:
Checking via IBinder is better